api-server/common/models/user.js

```diff
@@ -796,7 +796,16 @@ export default function(User) {
       ...user,
       about: showAbout ? about : '',
       calendar: showHeatMap ? calendar : {},
-      completedChallenges: showCerts && showTimeLine ? completedChallenges : [],
+      completedChallenges: (function() {
```

Comment:
```suggestion
      completedChallenges: (function() {
        return showTimeLine
          ? showCerts
            ? completedChallenges
            : completedChallenges.filter(({ challengeType }) => challengeType !== 7)
         : [];
```

Reply:
I tried to make a multi-line comment, but it did not look like it was going to work correctly if you committed it.  I can try again and can see what it looks like on your end.

Reply:
Yea, I tried a similar nested ternary - but prettier was complaining about it and wouldn't let me commit it.

Reply:
@ojeytonwilliams Why would this be?

Reply:
It's an eslint rule: [no-nested-ternary](https://eslint.org/docs/rules/no-nested-ternary).  

I'd go with

```js
if (showTimeLine) {
  return showCerts
    ? completedChallenges
    : completedChallenges.filter(
        challenge => challenge.challengeType !== 7
      );
} else {
  return [];
}
```

which it accepts and I think is fairly readable.

Reply:
At least destructure the `challengeType` from the challenge object in the filter callback function.